src/models/vec_model.jl

```diff
@@ -6,6 +6,10 @@ mutable struct VecModel{Tv <: AbstractVector} <: AbstractModel
     box_max::Tv
     init::Tv
     integer::BitVector
+    n_dim::Int
```

Comment:
Why do we need this? There is a `getdim` function.

Reply:
Oh I don't know that, I'll fix it